### ISSUE TITLE
Possibility to customize application tittle

### ISSUE THREAD

**Issue by damiansztankowski** [GoogleCloudPlatform/jit-access#172]
Hello,

Is there any chance to add possibility to change JIT tittle from " Just-in-Time Access" to custom one, provided at app.yaml file ? I know, that this is hardcoded and can be changed ( I did it ), but my deployment is referring to newest release always, so I don't want to use sed and change tittle each time, I update application. This is cosmetic changed imo, but I'm using different tittles for different environments and customers, so would be easier for our engineers to recognize for which env they granting access. 

cheers,
Damian

**Comment by jpassing** [comment]
I agree that this would be useful and it should be pretty straightforward to add a configuration variable for that.

**Comment by bschaatsbergen** [comment]
I'm happy to prep a fix for this.
